src/to-have-style.js

```diff
@@ -20,7 +20,7 @@ function isSubset(styles, computedStyle) {
     Object.entries(styles).every(
       ([prop, value]) =>
         computedStyle[prop] === value ||
-        computedStyle.getPropertyValue(prop.toLowerCase()) === value,
+        computedStyle.getPropertyValue(prop) === value,
```

Comment:
Wait, I think we may have a problem. I suspect that it was done this way because non-custom properties are case-insensitive. At least a quick check in the browser shows that setting `Display: none` hides the element.

Can you make it so that we keep the lower case transformation when the prop does not look like a custom css prop?

Reply:
Interesting, I'll look into it tomorrow!

Reply:
Good that there was a test case added with #148. Could have found out via `git blame`.